process/process.lua

```diff
@@ -359,6 +359,9 @@ function process.handle(msg, _)
     require('.eval')(ao)
   )
 
+  -- AR.IO PROCESS CODE
+  require(".src.init").init()
```

Comment:
nit: can probably move this down below _boot, just need to make sure globals gets loaded before we do anything in _boot

Reply:
moved to within _boot, with logs in https://github.com/ar-io/aos/pull/2/commits/024b223732e5264aaa837f56573a6e71e72fad93